prepare_deploy.py

```diff
@@ -135,8 +135,9 @@ def convert_notebooks(selected_nb_re=None):
 
         with open(html_filename, 'wb') as f:
             html_file_data = html_file_data.decode("utf-8")
-            html_file_data = html_file_data.replace("{title}",
-                                nb['metadata']['astropy-tutorials']['link_name'])
+            html_file_data = html_file_data.replace("{title}",nb['metadata']['astropy-tutorials']['link_name'])
+            html_file_data = html_file_data.replace("{abspageurl}","{}".format(cleanbase))
+			html_file_data = html_file_data.replace("{pageurl}","{}.html".format(cleanbase))
```

Comment:
Done ! It still shows ok in my local machine so I fixed it through Github's IDE...should be fine now ! Thanks for pointing out !